source/crud/read-operations/limit.txt

```diff
@@ -0,0 +1,78 @@
+====================================
+Limit the Number of Returned Results
+====================================
+
+.. default-domain:: mongodb
+
+Use limit to cap the number of documents that can be returned from a
+read operation. Limit functions as a cap on the maximum number of
+documents that the operation can return, but the operation can return
+a smaller number of documents if there are not enough documents present
+to reach the limit. If limit is used with the skip method, the skip will
+apply first and the cap will only apply to the documents left over after
+the skip.
+
+Consider a collection containing documents that describe books. To
+insert this data into a collection, run the following operation:
+
+.. code-block:: javascript
+
+   await collection.insertMany([
+     { "_id": 1, "name": "The Brothers Karamazov", "author": "Dostoyevsky", "length": 824 },
+     { "_id": 2, "name": "Les Misérables", "author": "Hugo", "length": 1462 },
+     { "_id": 3, "name": "Atlas Shrugged", "author": "Rand", "length": 1088 },
+     { "_id": 4, "name": "Infinite Jest", "author": "Wallace", "length": 1104 },
+     { "_id": 5, "name": "Cryptonomicon", "author": "Stephenson", "length": 918 },
+     { "_id": 6, "name": "A Dance With Dragons", "author": "Tolkein", "length": 1104 },
+   ]);
+
+Pass the following sort document to a read operation to ensure that the
+operation returns books with longer lengths before books with shorter
+lengths. Applying a limit of 3 to this operation causes this operation
+to return only the 3 longest books:
+
+.. code-block:: javascript
+   :emphasize-lines: 4
```

Comment:
can we highlight the `.sort(sort)` and `limit` lines as well?

Reply:
I would... but I can't do that any more after I move them to literalincludes. Put in a request with the platform team for that :)